watch_faces/BinarySystem/source/DataViews/DistanceView.mc

```diff
@@ -0,0 +1,47 @@
+using Toybox.Graphics as Gfx;
+using Toybox.ActivityMonitor as ActMon;
+using Toybox.Application as App;
+using Toybox.WatchUi as Ui;
+
+class DistanceView extends Ui.Drawable
+{
+    var showDistance = true;
+    var metricUnits = true;   //switch between metric and imperial true=metric / false=imperial
```

Comment:
according to the api the distance history is in cm
`distance ⇒ Number

Distance in cm

Returns:
(Number)
Since:
1.0.0`